### DISCUSSION TITLE
How to train my own instance segmentation dataset ?

### DISCUSSION THREAD

**Discussion by Mr-LiuDC** [jinfagang/yolov7#58] [Q&A]
How to train my own instance segmentation dataset? My dataset is coco format instance segmentation.

**Comment by pablodz** [comment]
x2

**Reply by Sahil028** [reply]
I suggest you try this link: https://www.youtube.com/watch?v=qej73NGDQfo 
I was able to do it following the same on colab, Trying to run inference on cpu, if you are able to crack it please let me know. Thanks :)

Just one thing that the installation for detectron2 is different in colab
